ztp/ztp-ran-policy-generator/ranGenTemplateExamples/site-du-sno-1-ranGen.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: ranPolicyGenerator/v1
+kind: RanGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "site-du-sno-1-policies"
+  namespace: "policy-template"
+  labels:
+    common: false
+    groupName: "N/A"
+    siteName: "site-du-sno-1"
+    mcp: worker-du
+sourceFiles:
+  - fileName: SriovNetwork
+    name: "sriov-nw-du-fh"
+    spec:
+      resourceName: du_fh
+      vlan: 140
+  - fileName: SriovNetworkNodePolicy
+    name: "sriov-nnp-du-fh"
+    spec:
+      deviceType: netdevice
+      isRdma: true
+      nicSelector:
+        pfNames: ["ens5f0"]
+      numVfs: 8
+      priority: 10
+      resourceName: du_fh
+  - fileName: SriovNetwork
+    name: "sriov-nw-du-mh"
+    spec:
+      resourceName: du_mh
+      vlan: 150
+  - fileName: SriovNetworkNodePolicy
+    name: "sriov-nnp-du-mh"
+    spec:
+      deviceType: vfio-pci
+      isRdma: false
+      nicSelector:
+        pfNames: ["ens7f0"]
+      numVfs: 8
+      priority: 10
+      resourceName: du_mh
+  - fileName: PerformanceProfile
+    name: "du-sno-1-PerfProfile"
+    spec:
+      # ex: overriding the additionalKernelArgs values exist in the source policy file PerformanceProfile
+      additionalKernelArgs:
```

Comment:
Why are we setting these ?

Reply:
Same as what we have at 5g-ref-nw below url. may @vitus133 can advise
https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/5g-ref-nw/profile-base/du-single/performance/performance-profile.yaml